src/fundus/parser/utility.py

```diff
@@ -378,51 +394,16 @@ def parse_title_from_root(root: lxml.html.HtmlElement) -> Optional[str]:
     return strip_nodes_to_text(title_node)
 
 
-def get_fundus_image_from_dict(json_dict: Dict[str, Any], domain: str) -> Optional[Image]:
-    """
-    Given a ld_json dictionary of type ImageObject, this helper function tries to parse the image and extract caption,
-    description and authors, if possible.
-    @param json_dict: Json Dictionary of type ImageObject
-    @param domain: Domain of the website hosting the image
-    @return: If the URL is valid, an Image object is returned
-    """
-    if (url := (json_dict.get("url") or json_dict.get("contentUrl"))) and isinstance(url, list):
-        valid_urls = list()
-        for url_str in url:
-            url_str = preprocess_url(url_str, domain)
-            if validators.url(url_str):
-                valid_urls.append(url_str)
-        if valid_urls:
-            return Image(
-                valid_urls,
-                description=json_dict.get("description"),
-                caption=json_dict.get("caption"),
-                author=generic_author_parsing(json_dict.get("author")),
-            )
-        return None
-
-    elif url:
-        url = preprocess_url(url, domain)
-        if validators.url(url):
-            return Image(
-                [url],
-                description=json_dict.get("description"),
-                caption=json_dict.get("caption"),
-                author=generic_author_parsing(json_dict.get("author")),
-            )
-    return None
-
-
 def preprocess_url(url: str, domain: str) -> str:
```

Comment:
I think this is leftover and can also be removed

Reply:
I wanted to keep it for the future. There may be a pint that this will come in handy.